src/Test.Utilities/CSharpCodeRefactoringVerifier`1.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.Testing;
+
+namespace Test.Utilities
+{
+    public static partial class CSharpCodeRefactoringVerifier<TRefactoring>
+        where TRefactoring : CodeRefactoringProvider, new()
+    {
+        /// <inheritdoc cref="CodeRefactoringVerifier{TCodeRefactoring, TTest, TVerifier}.VerifyRefactoringAsync(string, string)"/>
+        public static async Task VerifyRefactoringAsync(string source, string fixedSource)
+            => await VerifyRefactoringAsync(source, DiagnosticResult.EmptyDiagnosticResults, fixedSource);
+
+        /// <inheritdoc cref="CodeRefactoringVerifier{TCodeRefactoring, TTest, TVerifier}.VerifyRefactoringAsync(string, DiagnosticResult, string)"/>
+        public static async Task VerifyRefactoringAsync(string source, DiagnosticResult expected, string fixedSource)
+            => await VerifyRefactoringAsync(source, new[] { expected }, fixedSource);
+
+        /// <inheritdoc cref="CodeRefactoringVerifier{TCodeRefactoring, TTest, TVerifier}.VerifyRefactoringAsync(string, DiagnosticResult[], string)"/>
+        public static async Task VerifyRefactoringAsync(string source, DiagnosticResult[] expected, string fixedSource)
+        {
+            var val = new Test
+            {
+                TestCode = source,
+                FixedCode = fixedSource,
+            };
+
+            val.ExpectedDiagnostics.AddRange(expected);
+            await val.RunAsync(CancellationToken.None);
```

Comment:
There is no explicit call to the `CancellationToken` in the `CSharpCodeFixVerifier`, would it make sense to update it?

Reply:
Did not mean to include this parameter.

Reply:
Fixed in #3376